app-0-base-model/src/test/java/org/geovistory/toolbox/streams/base/model/processors/HasTypePropertyTest.java

```diff
@@ -39,19 +47,18 @@ void setup() {
         props.put(StreamsConfig.STATE_DIR_CONFIG, "/tmp/kafka-streams-test");
 
         var builderSingleton = new BuilderSingleton();
-        var avroSerdes = new AvroSerdes();
-        avroSerdes.QUARKUS_KAFKA_STREAMS_SCHEMA_REGISTRY_URL = MOCK_SCHEMA_REGISTRY_URL;
+
         var inputTopicNames = new InputTopicNames();
         var outputTopicNames = new OutputTopicNames();
-        var hasTypeProperty = new HasTypeProperty(avroSerdes, builderSingleton, inputTopicNames, outputTopicNames);
+        var hasTypeProperty = new HasTypeProperty(builderSingleton, inputTopicNames, outputTopicNames);
         hasTypeProperty.addProcessorsStandalone();
         var topology = builderSingleton.builder.build();
         testDriver = new TopologyTestDriver(topology, props);
 
         apiPropertyTopic = testDriver.createInputTopic(
                 hasTypeProperty.inApiProperty(),
-                avroSerdes.DfhApiPropertyKey().serializer(),
-                avroSerdes.DfhApiPropertyValue().serializer());
+                as.key().serializer(),
```

Comment:
@joschne in the test topic creation, this does not work, I don't know how to declare the topic line 33

Reply:
see commit a9b2d53